### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by Elkasitu**
This PR introduces the following changes:

* A fixup to the `osidb.acl` custom parameter being set on connection start
* Bump of the `Dockerfile`'s ubi8 version to 8.7 (latest)

### CODE REVIEW

File: config/settings_local.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
             "gssencmode": "disable",
             # this is a hack due to our inability to set a custom parameter either at
             # the database or role level in managed databases such as AWS RDS
-            "options": "-c osidb.acl=''",
+            "options": "-c osidb.acl=00000000-0000-0000-0000-000000000000",
```
Comment (osoukup) [inline config/settings_local.py:73]: I trust you this is correct but could you please explain what this value is? :smile:

**Review by osoukup — APPROVED** [review]
LGTM

File: config/settings_local.py (hunk 1 above)
Comment (Elkasitu) [reply config/settings_local.py:73]: It's the UUID equivalent of `NULL` or empty `''` [1]

[1] https://en.wikipedia.org/wiki/Universally_unique_identifier#Nil_UUID

Comment (osoukup) [reply config/settings_local.py:73]: Oh I see! Some Postgres magic :smile_cat:

**Review by JakubFrejlach — APPROVED** [review]

**Pull request merged**